core/src/test/java/org/elasticsearch/script/expression/ExpressionScriptTests.java

```diff
@@ -385,8 +388,63 @@ public void testStringSpecialValueVariable() throws Exception {
             message = e.toString();
         }
         assertThat(message + "should have contained ExpressionScriptExecutionException",
-                   message.contains("ExpressionScriptExecutionException"), equalTo(true));
+                message.contains("ExpressionScriptExecutionException"), equalTo(true));
         assertThat(message + "should have contained text variable error",
-                   message.contains("text variable"), equalTo(true));
+                message.contains("text variable"), equalTo(true));
+    }
+
+    public void testExecutableScripts() throws Exception {
```

Comment:
@colings86 I was hoping you could give me some help with this request for an aggregation test case.  Can you give me point to me the code you are using to update the variables for an aggregation using an expression outside of the search scope?  Thanks in advance.